test/core/rules/test_cases/L040.yml

```diff
@@ -0,0 +1,33 @@
+rule: L040
+
+test_1:
+  fail_str: |
+    SELECT
+        DISTINCT user_id,
+        list_id
+    FROM
+        safe_user
+  # The extra space before "user_id" is left over after the fix. In a full
```

Comment:
See my [question](https://github.com/sqlfluff/sqlfluff/pull/769/files#r576463716) on another PR about this kind of thing, where expected test results are ugly because they only run a single rule.